src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md

```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
   `paymentMethod`, when requesting payment.
 - `stripe.handleCardPayment`: Stripe Elements (card) is not needed if
   the default payment method is used.
+
+## Saving Payment Card with PaymentIntents payment flow
+
+Here's the description of complete call sequence on CheckoutPage.
+
+**Note: PaymentIntents flow needs transaction process change as
+described in doc:**
+
+[How to take Payment Intents into use](/guides/how-to-take-payment-intents-into-use/)
+
+### Initial data for Checkout:
+
+Check if the user has already saved a default payment method. Fetch
+currentUser entity with `stripeCustomer.defaultPaymentMethod`
+relationship. In FTW, we call a thunk function:
+[`fetchCurrentUser` in CheckoutPage.duck.js](https://github.com/sharetribe/flex-template-web/blob/master/src/containers/CheckoutPage/CheckoutPage.duck.js#L302).
+
+Behind the scenes, this is essentially the following call:
+
+```js
+=> sdk.currentUser.show({ include: ['stripeCustomer.defaultPaymentMethod'] })
+```
+
+### StripePaymentForm changes - show correct form fields:
+
+If there's a default payment method saved: show SavedCardDetails
+component. That component allows you to also select one-time payment
+instead (and optionally replace the current default payment method with
+this new payment card). If there's no default payment method saved,
+one-time payment form is shown on its own. One-time payment needs card
+details, card holder's name etc. and optional permission to save card
+details for future bookings.
+
+### Submit StripePaymentForm
+
+After submitting `StripePaymentForm`, there are up to 5 calls in
+sequence (to Flex and Stripe APIs):
+
+#### Step 1.
+
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', ...})
+```
+
+What happens behind the scene:
+
+- API creates PaymentIntent against Stripe API and returns
+  stripePaymentIntentClientSecret inside transaction's protectedData.
+- Booking is created at this step - so, availability management will
+  block dates for conflicting bookings.
+- Automatic expiration happens in 15 minutes, if process is not
+  transitioned to 'transition/confirm-payment' before that.
+- After this call, created transaction is saved to session storage in
+  FTW (or existing enquiry tx is updated).
+
+**When you save card details**, a new parameter needs to be passed if
+card details are saved at the same time: `setupPaymentMethodForSaving`
+
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', params: { listingId, bookingStart, bookingEnd, setupPaymentMethodForSaving: true }})
+```
+
+**When you use previously saved payment card**, the id of Stripe's
+payment method needs to be sent to Flex API as `paymentMethod`, when
+requesting payment.
+
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', params: { listingId, bookingStart, bookingEnd, paymentMethod: stripePaymentMethodId }})
```

Comment:
You could break pretty much all the examples into multiple lines for readability.

Reply:
I didn't notice that the extra "=>" string was a problem for Prettier.